src/main/java/com/taboola/backstage/model/media/campaigns/Campaign.java

```diff
@@ -181,11 +183,25 @@ public String getComments() {
     }
 
     public Date getStartDate() {
-        return startDate;
+        if (startDate != null) {
+            try {
+                return dateFormat.parse(startDate);
+            } catch (ParseException e) {
+                throw new RuntimeException(e);
+            }
+        }
+        return null;
     }
 
     public Date getEndDate() {
-        return endDate;
+        if (endDate != null) {
+            try {
+                return dateFormat.parse(endDate);
```

Comment:
The pattern is not correct it should be 'yyyy-MM-dd'

Reply:
Changed to 'yyyy-MM-dd'